dockerfiles/ci/buster/php-7.1/Dockerfile

```diff
@@ -72,7 +74,7 @@ RUN set -eux; \
         pecl install mcrypt-1.0.0; echo "extension=mcrypt.so" >> ${iniDir}/mcrypt.ini; \
         pecl install ast; echo "extension=ast.so" >> ${iniDir}/ast.ini; \
         yes 'no' | pecl install memcached; echo "extension=memcached.so" >> ${iniDir}/memcached.ini; \
-        pecl install mongodb; echo "extension=mongodb.so" >> ${iniDir}/mongodb.ini; \
+        pecl install mongodb-1.11.1; echo "extension=mongodb.so" >> ${iniDir}/mongodb.ini; \
```

Comment:
I would expect to see a similar line in other Dockerfiles -- it seems odd to have this in only 7.1, but not 7.0 nor 7.2.

Reply:
Ah, I see, it was previously pinned for 7.0.